ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+puts"how many emplyees will be processed?"
+number_of_employees = gets.chomp
+counter = 0 
+until counter == number_of_employees.to_i
+    puts "What is your name?"
+    name = gets.chomp
+    puts "How old are you?" 
+    age = gets.chomp
+
+    puts"What year were you born?"
+    year_born = gets.chomp
+    real_age = 2016 - year_born.to_i
+    if age == real_age
+        got_age_right = true
+    else got_age_right = false 
+    end
+
+    puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+    garlic_input = gets.chomp
+    if garlic_input == "yes"
+        likes_garlic = true
+    else likes_garlic = false
+    end
+
+    puts "Would you like to enroll in the company’s health insurance?"
+    insurance_input = gets.chomp
+    if insurance_input == "yes"
+        needs_insurance = true
+    else needs_insurance = false
+    end
+    valid_input = false
+    until valid_input 
```

Comment:
Collecting the allergy inputs and testing the data should be done in two separate loops.